locus.html

```diff
@@ -12,18 +12,147 @@
+          perception works and how we can use this understanding to maximise the
+          space that we occupy in our homes.
+        </p>
+
+        <h2>Out project</h2>
+        <p>Introducing our first official project</p>
+        <ul>
+          <li>
+            <h3>Project Livingroom</h3>
+            <p>
+              A simple guide to create the perfect livingroom for. Night Watc
+              works beautifully in a wide gamut of arrangements.
+            </p>
+          </li>
+          <li>
+            <h3>Project workspace</h3>
+            <p>
+              A simple guide to create the perfect livingro- om for . Night Watc
+              works beautifully in a wi- de gamut of arrangements.
+            </p>
+          </li>
+          <li>
+            <h3>Library</h3>
+            <p>
+              A simple guide to create the perfect livingro om for. Night Watc
+              works beautifully in a wide gamut of arrangements.
+            </p>
+          </li>
+        </ul>
+      </section>
+      <section>
+        <ul>
+          <li><b>105</b> project done</li>
+          <li><b>12</b> years of experience</li>
+          <li><b>10</b> big company partnership</li>
+        </ul>
+        <p>
+          "Not able to tell you how happy I am with interior design. Interior
+          design has completely surpassed our expectations. Interior design
+          saved my business. Interior design is worth much more than I paid."
+        </p>
+        <p>- Roscoe</p>
+      </section>
+      <section>
+        <h2>Contact Us</h2>
+        <p>Have any questions? Want help before getting started?</p>
+        <form action="#" method="GET">
+          <p>
+            <label for="name">Full name</label>
+            <input
+              id="name"
+              type="text"
+              name="customer_name"
+              placeholder="Full name"
+            />
+          </p>
+          <p>
+            <label for="email">Email</label>
+            <input
+              id="email"
+              type="text"
+              name="customer_email"
+              placeholder="Email address"
+            />
+          </p>
+          <p>
+            <label for="description">description</label>
+            <input
+              id="description"
+              type="text"
+              name="customer_description"
+              placeholder="Description"
+            />
+          </p>
+          <button type="submit">Send now</button>
+        </form>
+      </section>
+      <footer>
+        <h2>Locus</h2>
+        <ul>
+          <li>
+            <p>Interior design</p>
+            <a href="#"
+              >Jl.&nbspLaksda&nbspAdisucipto&nbspNo.51, Demangan,
+              Kec.&nbspDepok, Kota&nbspYogyakarta,
+              Daerah&nbspIstimewa&nbspYogyakarta&nbsp55282</a
+            >
+          </li>
+          <li>
+            <p>Perusahaan</p>
+            <a href="#">Tentang Kami</a>
```

Comment:
```suggestion
           <a href="#" aria-label="Tentang Kami">Tentang Kami</a>
```
так же для остальных